lib/web_ui/lib/src/engine/canvas_pool.dart

```diff
@@ -597,39 +606,117 @@ class _CanvasPool extends _SaveStackTracking {
     }
   }
 
+  /// Applys path to drawing context, preparing for fill and other operations.
+  ///
+  /// !Don't refactor _runPath/_runPathWithOffset. Latency sensitive
```

Comment:
nit: move the ! to the end of the sentence. Alternatively, begin with "WARNING:"

Reply:
Done.